packages/storybook/CHANGELOG.md

```diff
@@ -4,6 +4,13 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 
+v0.52.31
+------------------------------
+*July 07, 2022*
```

Comment:
Nit: 📆, and please remove leading `0`s from dates 😄

Reply:
Happy to - is that the general pattern everyone should be following or just a personal preference?

Reply:
It was the original pattern, so I'm just trying to keep it consistent 👍🏻